src/cache/precomputed_quantities.jl

```diff
@@ -602,16 +609,21 @@ NVTX.@annotate function set_precomputed_quantities!(Y, p, t)
         h_boundary_layer = f_b .* Fields.level(ᶜz, Spaces.nlevels(axes(Y.c)))
```

Comment:
```suggestion
        h_boundary_layer = f_b .* Fields.level(ᶜz, 1)
```
Let's talk about this.

Reply:
I'm leaving soon. But I think this makes more sense because otherwise h_boundary_layer will be the highest model layer if Ri is never smaller than Ri_c.